### PR TITLE
Type fixes

### PR DESCRIPTION
**Pull request by thewilkybarkid**
Fixing a few warnings from my IDE.

### CODE REVIEW

File: profiles/serializer/normalizer.py [hunk 1]
```diff
@@ -5,13 +5,13 @@
 
 
 @singledispatch
-def normalize(value: Any) -> str:
-    return str(value)
+def normalize(value: Any) -> Any:
```
Comment (thewilkybarkid) [inline profiles/serializer/normalizer.py]: No need to turn integers into strings. (This should really be string, numeric, dictionary, list, boolean or none.)

File: test/test_oauth2.py [hunk 2]
```diff
@@ -211,14 +211,14 @@ def test_it_exchanges(generate_random_string: MagicMock, test_client: FlaskClien
...
     with requests_mock.Mocker() as mocker:
-        def token_text(request: Request):
+        def token_text(request: PreparedRequest):
             return urlencode({
                 'client_id': 'server_client_id',
                 'client_secret': 'server_client_secret',
                 'redirect_uri': 'http://localhost/oauth2/check',
                 'grant_type': 'authorization_code',
                 'code': '1234',
-            }) == request.text
```
Comment (thewilkybarkid) [inline test/test_oauth2.py]: `text` came from `_RequestObjectProxy`, which is the actual type passed. It doesn't exist on `Request`.

File: profiles/serializer/normalizer.py (hunk 1 above)
Comment (thewilkybarkid) [reply profiles/serializer/normalizer.py]: Could introduce https://github.com/python/typing/issues/182?

Comment (seanwiseman) [reply profiles/serializer/normalizer.py]: I think its clear enough as is, these will only be used via the `json` module calls anyway so the expected values should be clear enough. If MyPy aren't pushing it any further at the moment I think we are fine. Interesting topic though.

**Pull request merged**